docs/parcels.md

```diff
@@ -0,0 +1,149 @@
+
+### Bootstrap
+This lifecycle function will be called once, right before the parcel is
+mounted for the first time.
+
+```js
+function bootstrap(props) {
+  return Promise
+    .resolve()
+    .then(() => {
+        // This is where you do one-time initialization
+        console.log('bootstrapped!')
+        });
+}
+```
+
+### Mount
+If the parcel is not mounted this lifecycle function is called when ever `mountParcel` is called. When 
+called, this function should create DOM elements, DOM event listeners, etc. to render content to the user.
+
+```js
+function mount(props) {
+  return Promise
+    .resolve()
+    .then(() => {
+        // This is where you tell a framework (e.g., React) to render some ui to the dom
+        console.log('mounted!')
+        });
+}
+```
+
+### Unmount
+This lifecycle function will be called whenever the parcel is mounted and one of the following cases is true:
+- `unmount()` is called
+- The parent parcel or application is unmounted
+When called, this function should clean up all DOM elements, DOM event listeners, leaked memory, globals, 
+observable subscriptions, etc. that were created at any point when the parcel was mounted.
+
+```js
+function unmount(props) {
+  return Promise
+    .resolve()
+    .then(() => {
+        // This is where you tell a framework (e.g., React) to unrender some ui from the dom
+        console.log('unmounted!');
+        });
+}
+```
+
+### Update (optional)
+This lifecycle function will be called whenever the parcel is mounted, and it needs to be updated.
+
+## Example use cases
+
+### Modals
+`App1` handles everything related to contacts (highly cohesive) but somewhere in `App2` we need to create a contact.
+We could do any number of things to share the functionality between application 1 and 2: 
+- If both are written in the same framework we could export/import components.
+- We could reimplement creating a contact (loss of cohesion)
+- We could use single-spa parcels.
+
+Exporting a parcel from `App1` that wraps the createContact modal component gives us the ability to share components and behavior across disparate frameworks, without losing application cohesion.
+App1 can export a modal as a single-spa parcel and App2 can import the parcel and use it easily. One major advantage is that in the below example
+the parcel/modal from App1 that is being used by App2 will also be unmounted, without unmounting/mounting of App1.
+
+```js
+// App1
+export const AddContactParcel = {
+  bootstrap: bootstrapFn,
+  mount: mountFn,
+  unmount: unmountFn,
+}
+
+// App2
+// get the parcel configuration in this case I'm using systemJS and react
+...
+componentDidMount() {
+  SystemJS.import('App1').then(App1 => {
+    const domElement = document.body
+    App2MountProps.mountParcel(App1.AddContactParcel, {domElement})
+  })
+}
+...
+
+```
+
+### Cross Library Components
```

Comment:
What is this section trying to accomplish? Are we trying to explain why we built parcels? Or how they compare to web components? What does the user gain from reading this section?